frontend/public/locales/en/public.json

```diff
@@ -1809,5 +1809,6 @@
   "default": "default",
   "OK": "OK",
   "Disabled": "Disabled",
-  "Enabled": "Enabled"
+  "Enabled": "Enabled",
+  "Installed": "Installed"
```

Comment:
Why are you changing public translations? 😕 

Seems like you missed a namespace somewhere.